test/values.ts

```diff
@@ -689,123 +689,101 @@ export const mockSimplexQuote = {
   supported_digital_currencies: ['CUSD', 'CELO'],
 }
 
-export const mockProviders: FetchProvidersOutput[] = [
+export const mockCicoQuotes: CicoQuote[] = [
```

Comment:
this mock data was transformed using [this method](https://github.com/valora-inc/cloud-functions/blob/main/src/endpoints/getCicoQuotes/extractQuotes.ts#L12) so test assertions can remain the same